frontend/src/old-pages/Configure/Components.tsx

```diff
@@ -373,176 +326,18 @@ function ArgEditor({path, i, multi, scriptIndex}: any) {
-    }
-
-    let knownExtension = findFirst(
-      knownExtensions,
-      (e: any) => e.path === scriptPath,
-    )
-    let scriptArgs = knownExtension
-      ? knownExtension.args.map((a: any) => `-${a.default || ''}`)
-      : []
-
-    let currentArgs = getState(path)
-    setState(path, [
-      ...currentArgs.slice(0, i),
-      baseScriptPath + scriptPath,
-      ...scriptArgs,
-      ...currentArgs.slice(end),
-    ])
-  }
-
-  const scriptToName = (script: any) => {
-    if (
-      script.startsWith(baseScriptPath) &&
-      knownScripts.includes(script.slice(baseScriptPath.length))
-    ) {
-      const path = script.slice(baseScriptPath.length)
-      const extension: Extension | undefined = findFirst(
-        knownExtensions,
-        e => e.path === path,
-      )
-      return extension ? extension.name : script
-    } else {
-      return script
-    }
-  }
-
-  return (
-    <div
-      style={{
-        display: 'flex',
-        flexDirection: 'row',
-        alignItems: 'center',
-        gap: '16px',
-      }}
-    >
-      <span style={{whiteSpace: 'nowrap'}}>Script:</span>
-      <Autosuggest
-        value={scriptToName(arg)}
-        onChange={({detail}) => {
-          if (detail.value !== arg && baseScriptPath + detail.value !== arg) {
-            if (knownScripts.includes(detail.value))
-              setKnownScript(detail.value)
-            else setState([...path, i], detail.value)
-          }
-        }}
-        enteredTextLabel={newValue => {
-          if (newValue !== arg) setState([...path, i], newValue)
-          return newValue
-        }}
-        ariaLabel="Script Selector"
-        placeholder="http://path/to/script"
-        empty="No matches found"
-        options={knownExtensions.map(({name, path, description}) => {
-          return {label: name, value: path, description: description}
-        })}
-      />
-      <Button onClick={remove}>
-        <span style={{whiteSpace: 'nowrap', marginRight: '40px'}}>Remove</span>
-      </Button>
-      <Button onClick={addArg}>
-        <span style={{whiteSpace: 'nowrap', marginRight: '40px'}}>Add Arg</span>
-      </Button>
-    </div>
-  )
-}
-
-function MultiRunnerEditor({path}: any) {
-  const data = useState(path) || []
-  const addScript = () => {
-    setState(path, [...data, ''])
-  }
-  let scriptIndex = -1
-  return (
-    <SpaceBetween direction="vertical" size="xs">
-      <Button onClick={addScript}>Add Script</Button>
-      {data.map((a: any, i: any) =>
-        a.length > 0 && a[0] === '-' ? (
-          <ArgEditor
-            key={`osa${i}`}
-            arg={a}
-            i={i}
-            path={path}
-            multi={true}
-            scriptIndex={scriptIndex}
-          />
-        ) : (
-          (() => {
-            scriptIndex = i
-            return <MultiRunnerScriptEditor key={`msa${i}`} path={path} i={i} />
-          })()
-        ),
-      )}
+      <Button onClick={remove}>{t('wizard.actions.remove')}</Button>
```

Comment:
todo: use `t` from the `useTranslation` hook, and not with a direct import

Reply:
Thanks, going to fix